resources/js/Pages/Results.vue

```diff
@@ -149,7 +150,8 @@
     interface MismatchDecision {
         id: number,
         item_id: string,
-        review_status: ReviewDecision
+        review_status: ReviewDecision,
+        old_status: ReviewDecision
```

Comment:
Works great, thanks!!. A nitpick: `previous_status` is a more appropriate name than `old_status`.